gnucash/ui/gnc-main-window.ui

```diff
@@ -0,0 +1,626 @@
+<?xml version="1.0"?>
+<interface>
+
+  <menu id="mainwin-menu">
+    <submenu>
+      <attribute name="label" translatable="yes">_File</attribute>
+      <attribute name="action">mainwin.FileAction</attribute>
+      <section>
+        <item>
+          <attribute name="label" translatable="no">FileNewAction</attribute>
+          <attribute name="action">gnc-plugin-basic-commands-actions.FileNewAction</attribute>
+        </item>
+        <item>
+          <attribute name="label" translatable="no">FileOpenAction</attribute>
+          <attribute name="action">gnc-plugin-basic-commands-actions.FileOpenAction</attribute>
+        </item>
+        <submenu>
+          <attribute name="label" translatable="no">FileImportAction</attribute>
+          <attribute name="action">mainwin.FileImportAction</attribute>
+          <item>
+            <attribute name="label" translatable="no">bi_importAction</attribute>
+            <attribute name="action">gnc-plugin-bi-import-actions.bi_importAction</attribute>
+          </item>
+          <item>
+            <attribute name="label" translatable="no">CsvImportAccountAction</attribute>
+            <attribute name="action">gnc-plugin-csv-import-actions.CsvImportAccountAction</attribute>
+          </item>
+          <item>
+            <attribute name="label" translatable="no">CsvImportTransAction</attribute>
+            <attribute name="action">gnc-plugin-csv-import-actions.CsvImportTransAction</attribute>
+          </item>
+          <item>
+            <attribute name="label" translatable="no">CsvImportPriceAction</attribute>
+            <attribute name="action">gnc-plugin-csv-import-actions.CsvImportPriceAction</attribute>
+          </item>
+          <item>
+            <attribute name="label" translatable="no">customer_importAction</attribute>
+            <attribute name="action">gnc-plugin-customer-import-actions.customer_importAction</attribute>
+          </item>
+          <item>
+            <attribute name="label" translatable="no">QIFImportAction</attribute>
+            <attribute name="action">gnc-plugin-qif-import-actions.QIFImportAction</attribute>
+          </item>
+          <item>
+            <attribute name="label" translatable="no">LogReplayAction</attribute>
+            <attribute name="action">gnc-plugin-log-replay-actions.LogReplayAction</attribute>
+          </item>
+          <item>
+            <attribute name="label" translatable="no">OfxImportAction</attribute>
+            <attribute name="action">gnc-plugin-ofx-actions.OfxImportAction</attribute>
+            <attribute name="hidden-when">action-disabled</attribute>
+          </item>
+          <item>
+            <attribute name="label" translatable="no">Mt940ImportAction</attribute>
```

Comment:
This breaks the modularity. GnuCash can be built without aqbanking support. If that's the case there shouldn't be traces from aqbanking in other parts, such as here in the UI files. The same may apply to ofx.

Reply:
I did it this way to avoid the complications of finding the location to add menu items and then add them. When built without aqbanking support they are all hidden. I will have a look at trying to do the addition in the three locations and do away with the already added menu items.

Reply:
Changed this to create the Aqbanking and Ofx menu entries from the associated plugin so no need to hide the menu entries when not built.

Reply:
(This was supposed to be my first comment for today, but I got side-tracked trying to understand the code)

I guess this works, but I wonder why you chose to hard-code these menu entries instead of using a ui file and using gtk_builder to load it as you do with the other ui files ?